Plugins/API/RainmeterAPI.cs

```diff
@@ -431,6 +442,49 @@ public void LogF(LogType type, string format, params Object[] args)
         {
             RmLog(this.m_Rm, type, string.Format(format, args));
         }
+
+        [DllImport("kernel32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
+        private static extern IntPtr LoadLibrary(string lpFileName);
+
+        [DllImport("kernel32.dll", SetLastError = true)]
+        private static extern IntPtr GetProcAddress(IntPtr hModule, string lpProcName);
+
+        [DllImport("Rainmeter.dll", CharSet = CharSet.Unicode, CallingConvention = CallingConvention.StdCall, EntryPoint = "RmGetOption")]

```

Comment:
@NSTechBytes Please confirm that you tested this. You marked this as resolved, but I want to you to explicitly say that it was tested with an old version of Rainmeter :)

I don't think this works like the C++ version because you are importing `RmGetOption` instead of accessing it through `GetProcAddress`.